content.js

```diff
@@ -211,17 +211,21 @@ const codes = {
 
 function replaceCasks()
 {
-  var casks = document.getElementsByClassName('caskNo');
+  var elementNames = ['caskNo', 'anotherClass', 'yetAnotherClass']; // Add more class names as needed
```

Comment:
remove the two additional example class names, leaving only one item in the elementNames